ledger/participant-state/kvutils/tools/src/main/scala/com/daml/ledger/participant/state/kvutils/tools/IntegrityCheck.scala

```diff
@@ -0,0 +1,145 @@
+// Copyright (c) 2019 The DAML Authors. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.participant.state.kvutils.tools
+
+import java.io.{DataInputStream, FileInputStream}
+import java.util.concurrent.TimeUnit
+
+import com.codahale.metrics
+import com.daml.ledger.participant.state.backport.TimeModel
+import com.daml.ledger.participant.state.kvutils.{DamlKvutils => Proto, _}
+import com.daml.ledger.participant.state.v1.{Configuration, Update}
+import com.digitalasset.daml.lf.data.Ref
+import com.digitalasset.daml.lf.engine.Engine
+
+import scala.collection.JavaConverters._
+import scala.util.Try
+
+object Helpers {
+  def time[T](act: () => T): (Long, T) = {
```

Comment:
Minor, but a by-name parameter is probably good enough here.
```suggestion
  def time[T](act: => T): (Long, T) = {
```